draft-ietf-core-coap-tcp-tls.md

```diff
@@ -224,35 +211,18 @@ BERT Option:
 {: vspace='0'}
 BERT Block:
 :	The payload of a CoAP message that is affected by a BERT Option in
-	descriptive usage (Section 2.1 of [I-D.ietf-core-block]).
+	descriptive usage (Section 2.1 of {{I-D.ietf-core-block}}).
 {: vspace='0'}
 # CoAP over TCP
 
-The request/response interaction model of CoAP TCP/TLS is similar to CoAP UDP.
-The primary differences are in the message layer. CoAP UDP supports optional
+# Architecture 
+
+The request/response interaction model of CoAP TCP/TLS is similar to CoAP UDP
+and the primary differences are in the message layer. CoAP UDP supports optional
 reliability by defining four types of messages: Confirmable, Non-confirmable,
 Acknowledgement, and Reset. TCP eliminates the need for the message layer
-to support reliability. As a result, message types are not defined in CoAP TCP/TLS.
-
-The protocol stack is illustrated in {{stack}}.
-
-~~~~
-        +--------------------------------+
-        |          Application           |
-        +--------------------------------+
-        +--------------------------------+
-        |  Requests/Responses/Signaling  |  CoAP (RFC 7252) and This Document
-        |--------------------------------|
-        |        Message Framing         |  This Document
-        +--------------------------------+
-        +---------------+          ^
-        |      TLS      |    or    |
-        +---------------+          v
-        +--------------------------------+
-        |              TCP               |
-        +--------------------------------+
-~~~~
-{: #stack title='The CoAP over TCP/TLS Protocol Stack' }

```

Comment:
We could keep this figure and just replace the TLS and TCP by "reliable transport".  It is certainly instructive, and has less noise than the previous one.